lib/chef/provider/aws_cache_replication_group.rb

```diff
@@ -0,0 +1,45 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsCacheReplicationGroup < Chef::Provisioning::AWSDriver::AWSProvider
+
+  protected
+
+  def create_aws_object
+    converge_by "create new Elasticache Replication Group #{new_resource.name} in #{region}" do
+      new_resource.driver.elasticache.create_replication_group(desired_options)
+    end
+  end
+
+  def update_aws_object(cache_replication_group)
+    Chef::Log.warn('Updating Elasticache Replication Groups is currently unsupported')
+  end
```

Comment:
The `describe_replication_group` method is so limiting in what it returns. It's almost impossible for us to tell if attributes have changed. It's also complicated because a replication group actually launches `n` cache clusters.